### PR TITLE
Address [objectors'] Proposal

### PR DESCRIPTION
**Pull request by torgo**
This text is intended to address the proposal from the [objectors]. The one change I have made from their proposal is to replace the word "address" with the word "counter" (which was the word originally used) to make it clear we need to work against "both deliberate and inadvertent attempts to mislead.").

### CODE REVIEW

**Review by chrisn — APPROVED** [review]

**Review by jyasskin — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -258,8 +256,9 @@ <h3 id="verify" data-export="" data-dfn-type="dfn">
   </h3>
   <p>
 Society relies on the integrity of public information.
-We have a responsibility to build web technologies to counter misinformation
-and to maintain the integrity of information for public good.
+We have a responsibility to build web technologies to counter both 
+deliberate and inadvertent attempts to mislead, and to maintain the 
+integrity of information for public good.
```
Comment (martinthomson) [inline index.html]:
```suggestion
We have a responsibility to build web technologies
to counter attempts to mislead,
deliberate or inadvertent,
and to maintain the integrity of information for public good.
```

Take it or leave it, the change is good.

**Review by martinthomson — APPROVED** [review]

**Review by hadleybeeman — APPROVED** [review]

**Comment by torgo** [comment]
I added this last commit to address the additional comment from the 2nd objector, which I had missed off.

File: index.html [hunk 2]
```diff
@@ -241,14 +241,11 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
   <p>
-We will create web technologies and platforms
-that encourage free expression,
-where that does not contravene other human rights.
+We will create web technologies and platforms that encourage free expression.
 Our work should not enable state censorship, <a data-cite="RFC7258#section-1">surveillance</a>
 or other practices that seek to limit this freedom.
 This principle must be balanced with respect for other human rights
 and does not imply that individual services on the web must therefore support all speech.
-(For example: hate speech, harassment or abuse may reasonably be denied a platform).
```
Comment (jyasskin) [inline index.html:251]: Could we keep this change to a different PR? I think I oppose it, and I wouldn't want the discussion to delay getting the rest in.

Comment (ylafon) [reply index.html:251]: For the sake of replying to each comments, I agree that this should be another PR (and be discussed by the TF)

**Review by jyasskin — APPROVED** [review]
